cluster-provision/gocli/cmd/run.go

```diff
@@ -384,9 +390,15 @@ func run(cmd *cobra.Command, args []string) (err error) {
 		}
 		volumes <- vol.Name
 
+		for i := 0; i < int(secondaryNics); i++ {
+			qemuArgs = fmt.Sprintf("%s -device virtio-net-pci,netdev=secondarynet%d -netdev tap,id=secondarynet%d,ifname=stap%d,script=no,downscript=no", qemuArgs, i, i, i)
```

Comment:
does this work with multiple nodes? please try.

Reply:
Yep, tested with 3 and eth1, eth2 and eth3 appears there.

Reply:
neat

Reply:
it doesn't work, it fails on:
```
--qemu-args "--qemu-args " -device virtio-net-pci,netdev=secondarynet0 -netdev tap,id=secondarynet0,ifname=stap0,script=no,downscript=no -device virtio-net-pci,netdev=secondarynet1 -netdev tap,id=secondarynet1,ifname=stap1,script=no,downscript=no" -device virtio-net-pci,netdev=secondarynet0 -netdev tap,id=secondarynet0,ifname=stap0,script=no,downscript=no -device virtio-net-pci,netdev=secondarynet1 -netdev tap,id=secondarynet1,ifname=stap1,script=no,downscript=no"
2019/06/21 08:25:19 Waiting for host: 192.168.66.102:22
provisioning node node02 failed
make: *** [Makefile:101: cluster-up] Error 1
```
I tried to utilize this change ([link](https://github.com/kubevirt/kubevirt/pull/2208/files#diff-629ce44207eb3be63928a0d226d0e672R10)), each lane fails.

Locally you can recreate that with k8s-1.13.3 and KUBEVIRT_NUM_NODES=2

Reply:
@mfranczy this is the error from container:
M IP in the guest network will be 192.168.66.102
VM hostname will be node02
+ echo 'SSH will be available on container port 2202.'
+ echo 'VNC will be available on container port 5902.'
+ echo 'VM MAC in the guest network will be 52:55:00:d1:55:02'
+ echo 'VM IP in the guest network will be 192.168.66.102'
+ echo 'VM hostname will be node02'
+ '[' '!' -e /dev/kvm ']'
+ exec qemu-system-x86_64 -enable-kvm -drive format=qcow2,file=/var/run/disk/disk.qcow2,if=virtio,cache=unsafe -device virtio-net-pci,netdev=network0,mac=52:55:00:d1:55:02 -netdev tap,id=network0,ifname=tap02,script=no,downscript=no -device virtio-rng-pci -vnc :02 -cpu host -m 5120M -smp 5 --qemu-args
qemu-system-x86_64: --qemu-args: invalid option

Will propose a PR in a few.

Reply:
Should be fixed with #110